context/response/responses.go

```diff
@@ -0,0 +1,53 @@
+package response
+
+import (
+	beecontext "github.com/astaxie/beego/context"
+)
+
+// JSON renders value to the response as JSON
+func JSON(value interface{}, encoding ...bool) Renderer {
```

Comment:
As context.Output already have a lot functions based on response. I think the seperate response package does not make sense here.